Src/GraphicLibs/Ncurses/NcursesLibrary.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+** EPITECH PROJECT, 2022
+** arcAAAAAAAAAAAAAAAAade
+** File description:
+** NcursesLibrary
+*/
+
+#include "NcursesLibrary.hpp"
+
+NcursesLibrary::NcursesLibrary()
+{
+    initscr();
+}
+
+NcursesLibrary::~NcursesLibrary()
+{
+    endwin();
+}
+
+bool NcursesLibrary::clearWindow()
+{
+    clear();
+    refresh();
+    return true;
+}
+
+bool NcursesLibrary::displayWindow()
+{
+    return true;
+}
+
+bool NcursesLibrary::drawCircle(std::pair<int, int> pos, int radius, Color color)
+{
+    return false;
+}
+
+bool NcursesLibrary::drawRect(std::pair<int, int> pos, int width, int height, Color color)
+{
+    init_pair(1, color.TB, color.TF);
+    attron(COLOR_PAIR(1));
+
+    move(pos.first, pos.second);
+    for (int i = 0; i < width; i++) {
+        for (int j = 0; j < height; j++) {
+            addch(' ');
+        }
+        move(pos.first, pos.second + i);
+    }
+
+    attroff(COLOR_PAIR(1));
+    reset_color_pairs();
+    return true;
+}
+
+bool NcursesLibrary::drawText(std::pair<int, int> pos, const std::string &content)
+{
+    move(pos.first, pos.second);
```

Comment:
Maybe pos.first - length /2 where length is the length of the string (this operation will center the text)